data/yam/agama/auto/template.jsonnet

```diff
@@ -0,0 +1,12 @@
+local lib = import 'lib/base.libsonnet';
```

Comment:
I guess we should import here all the libs and not have import libraries in libraries for simplicity.